src/main/scala/org/broadinstitute/hail/driver/SampleQC.scala

```diff
@@ -247,7 +251,12 @@ object SampleQC extends Command {
 
   def requiresVDS = true
 
-  def results(vds: VariantDataset): Map[String, SampleQCCombiner] = {
+  def results(vds: VariantDataset, branchingFactor: Option[Int]): Map[String, SampleQCCombiner] = {
+    val depth = branchingFactor
+      .map(b => (math.log(vds.nPartitions) / math.log(b) + 0.5).toInt.max(2))
```

Comment:
At some point we should look for zero here and do a regular aggregate.  Happy to leave it with `.max(1)` though